website/inject.js

```diff
@@ -185,7 +185,6 @@ async function injectGhStars () {
 
 async function injectMarkdown () {
   const sources = {
-    '.github/ISSUE_TEMPLATE/integration_help.md': 'src/_template/integration_help.md',
```

Comment:
is this unrelated?

Reply:
Seems https://github.com/transloadit/uppy/commit/a81b9485d32ac4fd6e44bafbd1e8c57aebe3de8b made it into here somehow 🤷